tests/test_core/conftest.py

```diff
@@ -0,0 +1,11 @@
+from etna.core.mixins import BaseMixin
+
+
+class Dummy(BaseMixin):
```

Comment:
May be we could name it somewhat more specific like: `BaseDummy` to be related to `BaseMixin`.